tests/integrational/ContenttypeTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests\integrational;
```

Comment:
Namespace should be uppercase and with proper prefix: `namespace Transfer\EzPlatform\Tests\Integration;`